traffic/configure-ios.md

```diff
@@ -10,27 +10,33 @@ previous_url: /knowledge-base/configure-ios, get-started/mobile-traffic/configur
 
 # Capturing and Inspecting iOS Traffic
 
-This article describes how use Fiddler Everywhere to capture and inspect traffic that comes from iOS devices and simulators.
+This article describes how to use Fiddler Everywhere to capture and inspect traffic that comes from iOS devices and simulators.
 
-To capture and inspect traffic on iOS devices, perform the following steps:
+To capture and inspect traffic on iOS devices or emulators, perform the following steps:
 
 1. [Provide the prerequisites](#prerequisites).
 1. [Configure Fiddler Everywhere](#configuring-fiddler-everywhere).
-1. [Configure the iOS device](#configuring-the-ios-device).
+1. [Configure iOS device](#configure-ios-device).
+1. [Configure iOS emulator](#configure-ios-device).
 1. [Inspecting the traffic](#inspecting-the-browser-traffic).
 
 ## Prerequisites
 
 - Install [Fiddler Everywhere v. 1.0.0 and above](https://www.telerik.com/download/fiddler-everywhere).
-- Install iOS.
+
+- Use an iOS device or an iOS emulator within the same local network as the Fiddler Everywhere host.
 
 ## Configuring Fiddler Everywhere
 
 1. Enable the remote connections of Fiddler Everywhere through **Settings** > **Connections** > **Allow remote computers to connect**.
 
 1. Check the IP address of the machine where Fiddler Everywhere is running. To obtain the IP address, for example, **ipconfig** on Windows or **ifconfig** on Linux, you can use the built-in OS tools or the Fiddler Everywhere popup status on the bottom-right part of the client. For demonstration purposes, let's assume that the local IP of the Fiddler Everywhere machine is 192.168.0.101.
 
-## Configuring the iOS Device
+1. Proceed with the iOS configuration. Note that [real iOS devices](#configuring-ios-device) have different configuration than [iOS emulators](#configuring-ios-emulator).
+
+## Configure iOS Device
+
+Follow the steps below to configure real iOS device to work alongside a Fiddler Everywhere host (check this section for [configuring iOS emulators](#configuring-ios-emulator)). Note that both devices have to be on the same network and discoverable. 
```

Comment:
To configure real iOS devices to work alongside a Fiddler Everywhere host, refer to the following steps. For more information on working with iOS emulators, refer to the section about [configuring iOS emulators](#configuring-the-ios-emulator). Both real iOS devices and iOS emulators have to be on the same network and be discoverable.